ludwig/features/feature_utils.py

```diff
@@ -116,11 +116,11 @@ def get_input_size_with_dependencies(
     """
     input_size_with_dependencies = combiner_output_size
     for feature_name in dependencies:
-        if other_output_features[feature_name].num_fc_layers:
+        if other_output_features[feature_name].decoder["num_fc_layers"]:
```

Comment:
I thought the decoder was schema-fied as well. Is that not the case?

Reply:
Ya you're right, this is a good catch. I think the dependencies aren't actually being acknowledged here which is why this isn't throwing an error. Looking into this.

Reply:
Wow thanks for catching that: this code would have failed if it had ever been called.  I'll update it and add a new test to `test_dependencies.py`